modules/navier_stokes/src/fv/kernels/INSFVMomentumAdvection.C

```diff
@@ -155,6 +155,40 @@ INSFVMomentumAdvection::INSFVMomentumAdvection(const InputParameters & params)
           id,
           " by flow_boundaries and the union of no_slip_wall_boundaries and slip_wall_boundaries.");
   }
+
+  if (getParam<bool>("force_boundary_execution"))
+    paramError("force_boundary_execution",
+               "Do not use the force_boundary_execution parameter to control execution of INSFV "
+               "advection objects");
```

Comment:
Yes it would. I expect an INSFV simulation, after this most recent round of changes, to not need/have `force_boundary_execution = true` anywhere. I guess that if you had some other finite volume physics present, this would be problematic. I know you are on the side of less verbosity in input files :smile: In the long (actually short) haul, I think it's best if we setup INSFV using an action, and then we don't have the concern you are implying

Reply:
Ok sounds good then